contracts/ServiceRegistry.sol

```diff
@@ -0,0 +1,179 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
try defaulting to `external` visibility where possible. see https://ethereum.stackexchange.com/questions/19380/external-vs-public-best-practices

Reply:
there is a pending PR on automation repo to change all `ServiceRegistry` visibility as well